.gitignore

```diff
@@ -7,4 +7,5 @@ obj
 packages
 Real-Time Corruptor/Output RTC
 RTC3 Launcher/.vs
-Real-Time Corruptor/BizHawk_RTC/.vs
\ No newline at end of file
+Real-Time Corruptor/BizHawk_RTC/.vs
+.vscode
```

Comment:
I have some .vscode tasks locally to help me out. I figured I shouldn't include them since only Visual Studio is "officially" supported

Reply:
I'm fine with having other common IDEs on the gitignore. Makes things easier.